ckanext/geocat/utils/xpath_utils.py

```diff
@@ -173,7 +173,7 @@ def xpath_get_rights_dict_form_rights_node(node):
             if value_locale:
                 rights_dict[locale.lower()] = _clean_string(value_locale[0])
         return rights_dict
-    except:
+    except Exception:
```

Comment:
Does it make sense to log the exception?

Reply:
To be honest, I'm not sure what kind of exception we could possibly be getting here. :-/ I don't think it's worth logging it, though, because when we use this rights dictionary, we actually only need it to have a valid rights name in one of the languages to get the correct rights id. (Here: https://github.com/opendata-swiss/ckanext-geocat/blob/master/ckanext/geocat/utils/csw_mapping.py#L399.)